src/api/api.js

```diff
@@ -1153,11 +1153,11 @@ export {
   getUserModelInfoApi,
   getBookByIdApi,
   getBookRecommendListApi,
-  getCategoriesApi,
+  getBookCategoriesApi,
```

Comment:
I have a little bit confused about this name. Since I thought this is a api that getting book's category. I am wondering "getBookListByCategoryIdApi()" is more readable? what do u think?